antlr-kotlin-examples-jvm/gradle

```diff
@@ -0,0 +1 @@
+../gradle
```

Comment:
should this be gradlew?

Reply:
I symlinked the gradle directory, gradlew and gradlew.bat to the parent gradlew installation, so we just need to update only one place.